blocks/api/utils.js

```diff
@@ -72,7 +72,20 @@ export function normalizeIconObject( icon ) {
 
 	if ( icon.background ) {
 		const tinyBgColor = tinycolor( icon.background );
```

Comment:
Can we change this abbreviation too? I know it's unrelated to your changes, but I think the code is easier to read with full words.